ubuntu/debian/tests/build

```diff
@@ -32,11 +32,11 @@ cmake_minimum_required(VERSION 3.5)
 
 project(sdf_test VERSION 1.0.0)
 
-find_package(sdformat13 13.0.0 REQUIRED)
+find_package(sdformat14 13.0.0 REQUIRED)
```

Comment:
```suggestion
find_package(sdformat14 14.0.0 REQUIRED)
```

Reply:
Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669